### DISCUSSION TITLE
[Unable to parse kube config: invalid configuration: unable to read certificate-authority][minikube]

### DISCUSSION THREAD

**Discussion by cham-d** [linkerd/linkerd2#8180] [Q&A]
Hello,
I am trying to set linkerd multicluster using **multiple minikubes** with **kvm2** driver.
I created two minikube: clusters east and west in the same network. They can ping each other.
I installed **metallb** on both, so loadbalancer services can get an external ip.
I followed the official tutorial for multiclustering over here: [https://linkerd.io/2.10/tasks/multicluster/](url)
,but when I run multicluster check, I get this:
![image](https://user-images.githubusercontent.com/55649286/161049511-5204964e-d57a-472f-9b94-a3ddfd0fedec.png)
The logs for the linkerd-service-mirror-east are:

`time="2022-03-31T11:56:17Z" level=info msg="Got updated link east: {Name:east Namespace:linkerd-multicluster TargetClusterName:east TargetClusterDomain:cluster.local TargetClusterLinkerdNamespace:linkerd ClusterCredentialsSecret:cluster-credentials-east GatewayAddress:192.168.59.110 GatewayPort:4143 GatewayIdentity:linkerd-gateway.linkerd-multicluster.serviceaccount.identity.linkerd.cluster.local ProbeSpec:ProbeSpec: {path: /ready, port: 4191, period: 3s} Selector:{MatchLabels:map[] MatchExpressions:[{Key:mirror.linkerd.io/exported Operator:Exists Values:[]}]}}"
time="2022-03-31T11:56:17Z" level=error msg="Unable to parse kube config: invalid configuration: unable to read certificate-authority /home/jim/.minikube/ca.crt for east due to open /home/jim/.minikube/ca.crt: no such file or directory"`

I see that it tries to read a ca.crt file, that doesn't exist. Actually, the path that it uses is from the host machine and I don't know why.
Any ideas?

**Comment by quangthe** [comment]
Same issue here. Have no ideas why it tried to access the minikube ca.crt file which is on the host (not visible in container)

**Comment by quangthe** [comment]
Cannot find any document mention multi-cluster with minikube. Maybe we can try with kind or k3d which mentioned in Linkder document.
